core/src/epicli/data/common/ansible/playbooks/roles/rabbitmq/tasks/install-packages-redhat.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Install packages
+  yum:
+    name:
+      - socat
```

Comment:
Do we really need it? :thinking:
```suggestion
```

Reply:
After grepping git repo I think socat could be completely removed... :thinking:
```
$ git grep socat
core/src/epicli/data/common/ansible/playbooks/roles/rabbitmq/tasks/install-packages-debian.yml:      - socat
core/src/epicli/data/common/ansible/playbooks/roles/rabbitmq/tasks/install-packages-redhat.yml:      - socat
core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt:socat
core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/requirements.txt:socat
core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/requirements.txt:socat
```

Reply:
Removed.